ravenframework/Optimizers/GeneticAlgorithm.py

```diff
@@ -254,16 +269,20 @@ def getInputSpecification(cls):
 
                                  b.    logistic: $fitness = \frac{1}{1+e^{a\times(obj-b)}}$.
 
-                                 c.    feasibleFirst: $fitness = \left\{\begin{matrix} -obj & g_j(x)\geq 0 \; \forall j \\ -obj_{worst}- \Sigma_{j=1}^{J}<g_j(x)> & otherwise \\ \end{matrix}\right.$""")
+                                 c.    feasibleFirst: $fitness = \left\{\begin{matrix} -obj & g_j(x)\geq 0 \; \forall j \\ -obj_{worst}- \Sigma_{j=1}^{J}<g_j(x)> & otherwise \\ \end{matrix}\right.$
+                                 
+                                 d.    hardConstraint: $fitness = the number of constraints violated.$
+                                 
+                                 """)
```

Comment:
if hardConstraint is not yet there, we shouldn't have it here. Usually, documentation happens after implementation. :)

Reply:
Any code lines related to hardConstraint is now commented.